tests/Test.ADAL.NET.Integration/ManagedUsercredentialTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Test.ADAL.NET.Common;
+using Test.ADAL.NET.Common.Mocks;
+using AuthenticationContext = Microsoft.IdentityModel.Clients.ActiveDirectory.AuthenticationContext;
+
+namespace Test.ADAL.NET.Integration
+{
+    [TestClass]
+    public class ManagedUsercredentialTests
+    {
+        [TestMethod]
+        [Description("Positive Test for AcquireToken with a user token already in the cache non-interactive")]
+        public async Task AcquireTokenWithUserTokenAlreadyInCachePositiveTestAsync()
```

Comment:
The name is incorrect. This test is not testing exactly that. Please refactor the test code to remove user2 call and mocks. Rename to emptycache...

Reply:
oh I see, wrong test case